lib/tapioca/sorbet_ext/generic_name_patch.rb

```diff
@@ -78,20 +78,22 @@ def name
   end
 
   module Utils
-    module CoercePatch
-      def coerce(val)
-        if val.is_a?(Tapioca::TypeVariableModule)
-          val.coerce_to_type_variable
-        elsif val.respond_to?(:__tapioca_override_type)
-          val.__tapioca_override_type
-        else
-          super
+    module Private
```

Comment:
I am afraid the fix for Tapioca is not as simple as changing the namespace and the method name, since after this change Tapioca will no longer work with any older version of Sorbet runtime.

We either need to require a newer version of `sorbet-runtime` as a runtime dependency, which I don't particularly like since it is too recent for most folks, or we need to make sure this code works properly both with the old Sorbet runtime code and the new one.

/cc @dirceu, @st0012 and @adisonlampert